gapis/api/cmd.go

```diff
@@ -58,6 +59,9 @@ type Cmd interface {
 	// Mutate mutates the State using the command. If the builder argument is
 	// not nil then it will call the replay function on the builder.
 	Mutate(context.Context, CmdID, *GlobalState, *builder.Builder) error
+
+	// Clone makes a shallow copy of this command.
+	CloneCmd(arena.Arena) Cmd
```

Comment:
In another branch I have a `clone()` that returns the concrete command type, and a `Clone()` that returns an `api.Cmd`. Would be nice if this matched.